src/pages/workspaces/workspace/notebooks/AnalysisLauncher.js

```diff
@@ -55,9 +55,24 @@ const AnalysisLauncher = _.flow(
     const { runtimeName, labels } = runtime || {}
     const status = getConvertedRuntimeStatus(runtime)
     const [busy, setBusy] = useState()
+    const [location, setLocation] = useState(defaultLocation)
     const { mode } = queryParams
     const toolLabel = getTool(analysisName)
     const iframeStyles = { height: '100%', width: '100%' }
```

Comment:
Would it be easier to notice the `useEffect` block with below?

```suggestion

    const iframeStyles = { height: '100%', width: '100%' }
```

Reply:
Yes, fixed.